src/Microsoft.Framework.PackageManager/Commands/WrapCommand.cs

```diff
@@ -47,7 +47,7 @@ public bool ExecuteCommand()
             var extension = Path.GetExtension(InputFilePath);
             if (string.Equals(extension, ".csproj", StringComparison.OrdinalIgnoreCase))
             {
-                if (PlatformHelper.IsMono)
+                if (!RuntimeEnvironmentHelper.IsWindows)
```

Comment:
IsMono was used incorrectly in most places because we were only doing stuff on Windows and on Mono, now that we are getting cross-plat CoreCLR support in we need to fix all of these incorrect IsMono's